app/src/main/java/net/squanchy/settings/SettingsFragment.java

```diff
@@ -124,6 +141,17 @@ private void onSignedOut() {
         );
     }
 
+    private boolean handleLocationPreferenceChange(Boolean isEnabling) {
+        if (isEnabling) {
+            navigator.toOnboardingForResult(OnboardingPage.LOCATION, TURN_LOCATION_ON_REQUEST_CODE);
+            return false;
```

Comment:
If we return true, the toggle animation begins while opening the onboarding screen, even if we are going to skip or dismiss the screen (then the toggle animates back off).
Are we okay with this side effect?